Source/WarpX.cpp

```diff
@@ -435,6 +438,8 @@ WarpX::ReadParameters ()
         // Read filter and fill IntVect filter_npass_each_dir with
         // proper size for AMREX_SPACEDIM
         pp.query("use_filter", use_filter);
+        pp.query("use_kspace_filter", use_kspace_filter);
+        pp.query("use_filter_compensation", use_filter_compensation);
```

Comment:
What happens if both `use_kspace_filter` and `use_filter` are true? Does it make sense to filter both in real space, when we call `SyncCurrent()`, and in Fourier space, or does it turn out to be redundant?

Reply:
If both flags are set, then both filters would be applied, which is probably not good. There should probably be an assert that both are not set, and possibly an assert that the physical space filter shouldn't be used with the spectral solver.